examples/grpc-unary-non-blocking/grpc_unary_non_blocking_client.out

```diff
@@ -1,11 +1,11 @@
-# To run the client and send request, create ballerina project and package
-# inside the project and include below files to package directory. Build the
-# project using build-in ballerina tool.
+# Create a Ballerina project and a package inside it. 
+# Add the following files inside the package directory. Build the
+# project using the built-in Ballerina tool.
```

Comment:
The following needs to be rephrased, if this built-in tool referred to here is the `build` command: "Build the project using the built-in Ballerina tool."

Reply:
@shavidissa, shall we make it like "Add the following files inside the package directory and build the project".
Thanks

Reply:
Please close this PR and do not merge. The changes are covered in this PR: https://github.com/ballerina-platform/ballerina-examples/pull/775